### PR TITLE
adds style scheme for rich text

### PR DESCRIPTION
**Pull request by txe**

### CODE REVIEW

File: src/ct/ct_treestore.cc [hunk 1]
```diff
@@ -178,6 +178,11 @@ Glib::RefPtr<Gsv::Buffer> CtTreeIter::get_node_text_buffer() const
     return rRetTextBuffer;
 }
 
+bool CtTreeIter::get_node_buffer_is_style_applied() const
+{
+    return !!(*this)->get_value(_pColumns->rColTextBuffer);
```
Comment (giuspen) [inline src/ct/ct_treestore.cc:183]: Is the double exclamation mark a typo?

Comment (txe) [reply src/ct/ct_treestore.cc:183]: No, it's an old trick to convert something to bool. First ! make it negative, second one reverted it back. For some reason '(*this)->get_value(_pColumns->rColTextBuffer)' didn't want to be converted to bool, so I had to use the trick.

Comment (txe) [reply src/ct/ct_treestore.cc:183]: Maybe I just need to compare the value with nullptr.

**Comment by giuspen** [comment]
I see, a static_cast<bool>() was more C++-ish, great work indeed anyway!

**Comment by txe** [comment]
Ah, right, always forgot about static_cast

**Pull request merged**